src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -10,24 +10,35 @@ public class WinningNumbers {
     private static final String SEPARATOR = ",";
     private static final String NULL_EXCEPTION_MESSGAE = "빈값이 들어왔습니다.";
     private static final String WINNING_NUMBER_EXCEPTION_MESSAGE = "당첨 번호는 6개만 입력해주세요.";
+    private static final String BONUSNUMBER_EXCEPTION = "이미 당첨번호에 포함 되어 있습니다.";
 
-    List<Integer> winningNumbers;
+    private final List<Integer> winningNumbers;
+    private final int bonusNumber;
 
-    public WinningNumbers(String winningNumbers) {
+    public WinningNumbers(List<Integer> winningNumbers) {
         validationBlank(winningNumbers);
-        this.winningNumbers = winningNumbersMake(winningNumbers);
+        validationNumbersSizeCheck(winningNumbers);
+        this.winningNumbers = winningNumbers;
+        this.bonusNumber = 0;
```

Comment:
```suggestion
        this(winningNumbers, 0);
```

인스턴스 변수의 초기화는 주 생성자에서 담당하도록 구현한다.